src/components/website-management/website-form-components/customization/customization-components/CustSelectBox.js

```diff
@@ -0,0 +1,25 @@
+import { FormControl, InputLabel, MenuItem, Select } from "@mui/material";
+const CustSelectBox = (props) => {
+  const { name, fun, items, width, selectedTemplate } = props;
```

Comment:
What does `fun` mean here ;) Shall we use meaningful names for the variables?

Reply:
fun = function in here